aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -337,7 +337,7 @@ If you need to transform *web.config* on publish (for example, set environment v
 * <xref:host-and-deploy/web-farm>
 * [Tutorial: Connect to SQL Database from .NET App Service without secrets using a managed identity](/azure/app-service/tutorial-connect-msi-sql-database?tabs=efcore%2Cdotnetcore)
 
-Azure App Service on Windows Server uses [Internet Information Services (IIS)](https://www.iis.net/). The following topics pertain to the underlying IIS technology:
+Azure App Service on Windows Server uses [Internet Information Services (IIS)](https://www.iis.net/). [Kestrel and YARP](https://devblogs.microsoft.com/dotnet/bringing-kestrel-and-yarp-to-azure-app-services/) on the front end provides the load balancer. The following topics pertain to the underlying IIS technology:
```

Comment:
> If we want to talk about Front Ends, then adding a link to the blog is fine, but in general FE are not something that customers have any control when hosting theirs Apps on App Service and is more of an implementation detail of the platform.

@btardif 
OK to add this sentence which clears up the confusion from a quick read of your blog?